### ISSUE TITLE
Ubuntu 18.04 old package version

### ISSUE THREAD

**Issue by napei** [tonsky/FiraCode#673]
Currently, under [https://packages.debian.org/sid/fonts-firacode](https://packages.debian.org/sid/fonts-firacode) the latest vesion is 1.204-2 but the latest version on github is 1.206.

The version `1.204-2` never existed on github with that versioning format so I'm confused as to what version debian actually gets.

**Comment by NoahTheDuke** [comment]
It’s possible Debian builds their own version.

**Comment by Somebody62** [comment]
Shouldn't you report this to Debian instead? It seems this is their fault, not providing the most up-to-date version.

**Comment by napei** [comment]
@Somebody62 Yes, but @tonsky owns the package so that's his responsibility. I'm merely notifying him.

**Comment by tonsky** [comment]
I certainly do not. I don’t even have a linux
<img width="306" alt="screenshot 2018-10-21 at 13 11 26" src="https://user-images.githubusercontent.com/285292/47265597-f3ca4280-d532-11e8-8918-412ef92e2eec.png">

**Comment by napei** [comment]
I meant you maintain fira code not the debian release. But seeing as you don't manage the release on debian that's okay then.
